.github/workflows/mingw-w64-tiledb/PKGBUILD

```diff
@@ -0,0 +1,52 @@
+_realname=tiledb
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=2.9000
+pkgrel=1
+pkgdesc="Storage management library for sparse and dense array data (mingw-w64)"
+arch=("any")
+url="https://tiledb.com/"
+license=("MIT")
+depends=("${MINGW_PACKAGE_PREFIX}-lz4"
+         "${MINGW_PACKAGE_PREFIX}-aws-sdk-cpp"
+         "${MINGW_PACKAGE_PREFIX}-bzip2"
+         "${MINGW_PACKAGE_PREFIX}-zlib"
+         "${MINGW_PACKAGE_PREFIX}-zstd")
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake"
+             "${MINGW_PACKAGE_PREFIX}-gcc"
+             "${MINGW_PACKAGE_PREFIX}-curl")
+options=("staticlibs" "strip")
+source_dir="$TILEDB_HOME"
+
+build() {
+  [[ -d ${source_dir}/build-${MINGW_CHOST} ]] && rm -rf ${source_dir}/build-${MINGW_CHOST}
+  mkdir -p ${source_dir}/build-${MINGW_CHOST} && cd ${source_dir}/build-${MINGW_CHOST}
+
+  if [ "$CARCH" == "i686" ]; then
+  export CFLAGS="-mfpmath=sse -msse2"
+  export CXXFLAGS="-mfpmath=sse -msse2"
+  fi
+
+  export MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX="
+  ${MINGW_PREFIX}/bin/cmake.exe \
+    -G"MSYS Makefiles" \
+    -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
+    -DBUILD_SHARED_LIBS=OFF \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DTILEDB_STATIC=ON \
+    -DTILEDB_TBB=OFF \
+    -DTILEDB_S3=ON \
+    ..
+  make
+  make -C tiledb
+}
+
+package() {
+  cd ${source_dir}/build-${MINGW_CHOST}
+  make DESTDIR="${pkgdir}" -C tiledb install
+}
+
+check (){
+  cd ${source_dir}/build-${MINGW_CHOST}
+  #make check
```

Comment:
Does this pass when enabled? We should enable it here, although will need some follow-up work to enable S3 testing.

Reply:
No, building the tests currently fails with gcc due to deprecation warnings, but I don't think this is related to mingw.

Reply:
I see, ok good point.